src/site/content/es/fast/lazy-loading-images/index.md

```diff
@@ -5,7 +5,7 @@ authors:
   - jlwagner
   - rachelandrew
 date: 2019-08-16
-updated: 2020-06-09
+updated: 2022-08-11
```

Comment:
I dunno how translations work, but in general I've avoided updating the date when I can't make the complete changes to pull it up to speed.

I figured that it would be more obvious it was still out of date then compared to English (both for readers AND translators).

WYDT since you're only doing some of the changes here (removing IE stuff, but not adding the LCP stuff)?

Reply:
I used Google Translate to isolate the stuff about Internet Explorer. I felt that guidance was worth removing for all languages, and updating `updated` seems like a good move here.

As for updating translations, we outsource that I think, but I don't know what the pipeline is for that process.

Reply:
Yeah I kind made up that rule for not updating - and usually cause I was only doing simple edits or link corrections that arguably weren't really worthy of incrementing the date anyway. Yours is a bit more substantial as removing large chunks.

Cool will approve. Let us know if you ever find out what the pipeline is.